src/injections/css/bug1570328-developer-apple.com-transform-scale.css

```diff
@@ -0,0 +1,18 @@
+/**
+ * developer.apple.com - content of the page is shifted to the left
+ * Bug #1570328 - https://bugzilla.mozilla.org/show_bug.cgi?id=1570328
+ * WebCompat issue #4070 - https://webcompat.com/issues/4070
+ *
+ * The site is relying on zoom property which is not supported by Mozilla,
+ * see https://bugzilla.mozilla.org/show_bug.cgi?id=390936. Adding a combination
+ * of transform: scale(1.4), transform-origin and width fixes the issue
+ */
+@media only screen and (min-device-width: 320px) and (max-device-width: 980px),
+  (min-device-width: 1024px) and (max-device-width: 1024px) and (min-device-height: 1366px) and (max-device-height: 1366px) and (min-width: 320px) and (max-width: 980px),
+  (min-device-width: 1024px) and (max-device-width: 1024px) and (min-device-height: 1366px) and(max-device-height: 1366px) and (min-width: 320px) and (max-width: 980px) {
```

Comment:
I have no idea why prettier does not care about this, but `and(max-device-height: 1366px)` is missing a space after the `and`.

Reply:
so we even need line 12? Isn't it exactly the same as the previous line? What am I missing? :)

Reply:
Ah, good point. I've just copied it from the website's style sheet. I'll remove the last line